### ISSUE TITLE
Can this be used for MultiWorlds?

### ISSUE THREAD

**Issue by Merith-TK** [Emafire003/SeedLight-Riftways#1]
My question is, can this be used to have multiple worlds on the server?

**Comment by Emafire003** [comment]
Not really, you would need multiple instances of a server to do this with this mod. What you're looking for sounds more like the Multiverse Core plugin or stuff like that

**Comment by Merith-TK** [comment]
Fabric server, cant really use Multiverse unfortunately, 

would it be possible for you to add support for multiple worlds on the server? would probably fall under "dimensions" where you could set an tag to be `dimension/minecraft:the_nether` for example

**Comment by Emafire003** [comment]
I think I could, but i'm really busy lately no idea when I could actually implement it sorry. In the mean time I think there is something like it somewhere for fabric too tho
